curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64ff068e0426eb288874ed79.md

```diff
@@ -0,0 +1,406 @@
+---
+id: 64ff068e0426eb288874ed79
+title: Step 52
+challengeType: 0
+dashedName: step-52
+---
+
+# --description--
+
+You can use `localStorage.removeItem()` to remove an item and `localStorage.clear()` to clear all items in the local storage.
```

Comment:
```suggestion
You can use `localStorage.removeItem()` to remove a specific item and `localStorage.clear()` to clear all items in the local storage.
```

Reply:
@Ksound22

It looks like this conversation hasn't been resolved yet